frontend/packages/topology/integration-tests/support/step-definitions/topology/chart-view.ts

```diff
@@ -122,3 +125,59 @@ When('user clicks on {string} option', (resourceType: string) => {
 Then('user can see only the {string} workload', (workload: string) => {
   cy.get(typeOfWorkload(workload)).should('be.visible');
 });
+
+When('user selects {string} option in filter menu', (searchType: string) => {
+  cy.get('#toggle-id').click();
+  cy.get("[role='menu']")
+    .find('li')
+    .contains(searchType)
+    .should('be.visible')
+    .click();
+});
+
+When('user searches for label {string}', (search: string) => {
+  cy.get(topologyPO.search)
+    .clear()
+    .type(search);
+  cy.get('.co-suggestion-box__suggestions')
+    .find('button')
+    .contains(search)
+    .should('be.visible')
+    .first()
+    .click();
+});
+
+Then('user can see the workload {string} visible', (workload: string) => {
+  cy.get(topologyPO.highlightNode).should('be.visible');
+  cy.get(topologyPO.highlightNode).contains(workload);
```

Comment:
I think this should be an assertion, rather than element lookup when there is no actual interaction with the element.

```typescript
cy.get(topologyPO.highlightNode).should('contain', workload);
```

Reply:
Sorry, missed it